client/my-sites/plans-features-main/index.tsx

```diff
@@ -727,6 +731,12 @@ const PlansFeaturesMain = ( {
 
 	const isPlansGridReady = ! isLoadingGridPlans && ! resolvedSubdomainName.isLoading;
 
+	const enablePlanTypeSelectorStickyBehavior = isMobile() && showPlanTypeSelectorDropdown;
```

Comment:
**Note:**
TL;DR I'm using javascript instead of CSS to handle logic for mobile screens because we're outside of the `plans-grid` shared package and because it noticeably simplifies logic.

We want sticky behavior for _only_ for mobile screens.

We chatted about, where possible, keeping desktop vs. tablet vs. mobile logic to CSS media queries in the `plans-grid` for the sake of convention. To avoid issues, I move mobile "definitions" into `plans-features-main` by using `isMobile` from the `'@automattic/viewport'` package.

I do this because relying on media queries for our use case ends up making feature logic more complex. We would effectively replicate behavior with class names and CSS that the `StickyContainer` already supports through programmatic javascript: the ability to enable and disable sticky behavior under different conditions. Here's an existing example currently in use in the comparison grid https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plans-grid/components/comparison-grid/index.tsx#L1094